### PR TITLE
New Jersey TANF Maximum Benefits Payment

### LINKED ISSUE #2148: New Jersey Maximum Benefits Payment
https://casetext.com/regulation/new-jersey-administrative-code/title-10-human-services/chapter-90-work-first-new-jersey-program/subchapter-3-financial-eligibility-income-resources-benefits/section-1090-33-wfnjtanf-initial-allowable-maximum-income-and-maximum-benefit-payment-levels-schedules-i-and-ii

### PR DESCRIPTION
**Pull request by xudonglai0426**
Fixes #2148

Co-authored-by: VioletChen0916 <VioletChen0916@users.noreply.github.com>

### Summary
:sparkles::pencil2::link:

This pull request adds a new parameter and variable for the New Jersey TANF maximum benefit, updates the Maine income tax deduction parameters and tests, and fixes some formatting issues in the changelog and other files.

> _`TANF` parameter_
> _New Jersey joins the model_
> _Autumn of updates_

### Walkthrough
*  Add a new variable and parameters for New Jersey TANF maximum benefit ([link](https://github.com/PolicyEngine/policyengine-us/pull/2150/files?diff=unified&w=0#diff-2d25a440af9670ebbeb80124049924dd54019b67d9545761d37882f3ff9c7fdbR1-R12), [link](https://github.com/PolicyEngine/policyengine-us/pull/2150/files?diff=unified&w=0#diff-aa33c7997ca138e744c891b38b96a1711fba68f738d1510e5741886693f2283fR1-R28), [link](https://github.com/PolicyEngine/policyengine-us/pull/2150/files?diff=unified&w=0#diff-0e95f0a924636185eb82faee914e5af43a03cb531e9bfc6a015d18d514c68321R1-R27))
* Update the parameters and references for Maine income tax deductions ([link](https://github.com/PolicyEngine/policyengine-us/pull/2150/files?diff=unified&w=0#diff-b12a552d7a4cf95232ffffef6fcaff276a1537b2b7906b3a06a1288cacc6a6d2L19-R19), [link](https://github.com/PolicyEngine/policyengine-us/pull/2150/files?diff=unified&w=0#diff-06bb5c11c5237a5dbed498e2a362894b1520a792166f70bd1bea29ddfa263addL20-R20))
  - Fix typos in the test files `me_deduction.yaml` and `me_standard_deduction.yaml` where the output values were missing decimal points ([link](https://github.com/PolicyEngine/policyengine-us/pull/2150/files?diff=unified&w=0#diff-01a4efd3db148ba9d5c3ae3e62393bb5cfb6219ccc086ee60e25a53c1035c47eL45-R45), [link](https://github.com/PolicyEngine/policyengine-us/pull/2150/files?diff=unified&w=0#diff-25f9ab7b10d5cb5d446002908282f15aa7fab0d724c0c32b1aeb1633e2d9eed4L39-R39))
* Add a new entry to the changelog file `changelog_entry.yaml` indicating the minor version bump, the new parameter for New Jersey TANF, and the formatting fixes ([link](https://github.com/PolicyEngine/policyengine-us/pull/2150/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8R1-R6))

### CODE REVIEW

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    fixed:
```
Comment (MaxGhenis) [inline changelog_entry.yaml:3]:
```suggestion
    added:
```

File: changelog_entry.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+    - Update values for NJ TANF maximum benefit
```
Comment (MaxGhenis) [inline changelog_entry.yaml:4]: It wasn't there previously to update right?
```suggestion
    - NJ TANF maximum benefit.
```

File: policyengine_us/parameters/gov/states/nj/njdhs/tanf/maximum_benefit/additional.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+description: New Jersey limits its TANF program to households with this maximum benefit for people beyond size.
+
+metadata:
+  unit: currency-USD
+  period: month
+  label: New Nersey TANF monthly maximum benefit per additional person
```
[the remark this answers is not in the document]
Comment (xudonglai0426) [reply policyengine_us/parameters/gov/states/nj/njdhs/tanf/maximum_benefit/additional.yaml]: We do find a state plan for new jersey tanf (2021-2023). Here is the link: https://www.nj.gov/humanservices/dfd/programs/workfirstnj/tanf_2021_23_st_plan.pdf#page=63
However, the values of maximum benefit are different from those values on State Code. After looking at the state plan (2018-2020), we find the values are also different. Therefore, we assume the new jersey government adjusts the values maybe every 2 year. We are trying to find some documents that describe the adjustment but haven't got any result yet.

**Comment by xudonglai0426** [comment]
We find a state plan for new jersey tanf (2021-2023). Here is the link: https://www.nj.gov/humanservices/dfd/programs/workfirstnj/tanf_2021_23_st_plan.pdf#page=63
However, the values of maximum benefit are different from those values on State Code. After looking at the state plan (2018-2020): https://www.state.nj.us/humanservices/providers/grants/public/publicnoticefiles/FINAL%20Draft%20NJ%20TANF%20State%20Plan%20FFY2018-2020.pdf#page=62, we find the values are also different. Therefore, we assume the new jersey government adjusts the values maybe every 2 year when new state plan published and we are using the latest state plan(2021-2023) as reference.

File: policyengine_us/parameters/gov/states/nj/njdhs/tanf/maximum_benefit/additional.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+description: New Jersey limits its TANF program to households with this maximum benefit for people beyond size.
+
+metadata:
+  unit: currency-USD
+  period: month
+  label: New Jersey TANF monthly maximum benefit per additional person
+  reference:
+    - title: New Jersey Administrative Code | § 10:90-3.3 WFNJ/TANF-initial allowable maximum income and maximum benefit payment levels (Schedules I and II)
+      href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=977adb3e-e3a6-4eba-8d54-e69ee35f7dcd&nodeid=AAOAERAAEAAD&nodepath=%2FROOT%2FAAO%2FAAOAER%2FAAOAERAAE%2FAAOAERAAEAAD&level=4&haschildren=&populated=false&title=%C2%A7+10%3A90-3.3+WFNJ%2FTANF-initial+allowable+maximum+income+and+maximum+benefit+payment+levels+(Schedules+I+and+II)&config=00JAA1YTg5OGJlYi04MTI4LTRlNjQtYTc4Yi03NTQxN2E5NmE0ZjQKAFBvZENhdGFsb2ftaXPxZTR7bRPtX1Jok9kz&pddocfullpath=%2Fshared%2Fdocument%2Fadministrative-codes%2Furn%3AcontentItem%3A5XKV-PWD1-JTGH-B3FM-00008-00&ecomp=8gf5kkk&prid=039b2ed1-6268-497c-a43f-3e4c508cdc00
+    - title: New Jersey State Plan for TANF(2021-2023)
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/nj/njdhs/tanf/maximum_benefit/additional.yaml:10]:
```suggestion
    # State plan has updated values.
    - title: New Jersey State Plan for TANF(2021-2023)
```

File: policyengine_us/parameters/gov/states/nj/njdhs/tanf/maximum_benefit/additional.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+description: New Jersey limits its TANF program to households with this maximum benefit for people beyond size.
+
+metadata:
+  unit: currency-USD
+  period: month
+  label: New Jersey TANF monthly maximum benefit per additional person
+  reference:
+    - title: New Jersey Administrative Code | § 10:90-3.3 WFNJ/TANF-initial allowable maximum income and maximum benefit payment levels (Schedules I and II)
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/nj/njdhs/tanf/maximum_benefit/additional.yaml:8]:
```suggestion
    # Administrative code has old values, apparently without inflation adjustments.
    - title: New Jersey Administrative Code | § 10:90-3.3 WFNJ/TANF-initial allowable maximum income and maximum benefit payment levels (Schedules I and II)
```

File: policyengine_us/parameters/gov/states/nj/njdhs/tanf/maximum_benefit/main.yaml [hunk 6]
```diff
@@ -0,0 +1,30 @@
+description: New Jersey limits its TANF program to households with this maximum benefits, by household size.
+
+metadata:
+  unit: currency-USD
+  period: month
+  breakdown:
+    - range(1, 9)
+  label: New Jersey TANF monthly maximum benefit
+  reference:
+    - title: New Jersey Administrative Code | § 10:90-3.3 WFNJ/TANF-initial allowable maximum income and maximum benefit payment levels (Schedules I and II)
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/nj/njdhs/tanf/maximum_benefit/main.yaml:10]:
```suggestion
    # Administrative code has old values, apparently without inflation adjustments.
    - title: New Jersey Administrative Code | § 10:90-3.3 WFNJ/TANF-initial allowable maximum income and maximum benefit payment levels (Schedules I and II)
```

File: policyengine_us/parameters/gov/states/nj/njdhs/tanf/maximum_benefit/main.yaml [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+      href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=977adb3e-e3a6-4eba-8d54-e69ee35f7dcd&nodeid=AAOAERAAEAAD&nodepath=%2FROOT%2FAAO%2FAAOAER%2FAAOAERAAE%2FAAOAERAAEAAD&level=4&haschildren=&populated=false&title=%C2%A7+10%3A90-3.3+WFNJ%2FTANF-initial+allowable+maximum+income+and+maximum+benefit+payment+levels+(Schedules+I+and+II)&config=00JAA1YTg5OGJlYi04MTI4LTRlNjQtYTc4Yi03NTQxN2E5NmE0ZjQKAFBvZENhdGFsb2ftaXPxZTR7bRPtX1Jok9kz&pddocfullpath=%2Fshared%2Fdocument%2Fadministrative-codes%2Furn%3AcontentItem%3A5XKV-PWD1-JTGH-B3FM-00008-00&ecomp=8gf5kkk&prid=039b2ed1-6268-497c-a43f-3e4c508cdc00
+    - title: New Jersey State Plan for TANF(2021-2023)
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/nj/njdhs/tanf/maximum_benefit/main.yaml:12]:
```suggestion
    # State plan has updated values.
    - title: New Jersey State Plan for TANF(2021-2023)
```

**Pull request merged**